index.html

```diff
@@ -275,6 +275,10 @@
         }
         i.main-logo {
           margin-top: 160px;
+          width: 300px;
+          height: 160px;
+          background-size: 290px 134px;
```

Comment:
I don't understand why `background-size` and `width`/`height` are different? Shouldn't this be

```css
width: 290px;
height: 134px;
```